lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -56,7 +56,7 @@ def attempted(self):
             "SubsectionGrade not fully populated yet.  Call init_from_structure or init_from_model "
             "before use."
         )
-        return self.all_total.attempted
+        return self.all_total.first_attempted is not None
```

Comment:
Who uses this property right now?  Should we just change it to return the timestamp instead?